articles/api/overview.md

```diff
@@ -0,0 +1,83 @@
+
+> For more information on tokens please refer to: [Tokens used by Auth0](/tokens).
+
+## How to configure an API in Auth0
+
+If this is your first API, you will have to enable the API functionality first.
+
+Navigate to your [Account Advanced Settings](${manage_url}/#/account/advanced), scroll down to the *Settings* section and toggle the **Enable APIs Section** switch.
+
+![Enable APIs Section](/media/articles/api/overview/enable-api-section.png)
+
+Click on the [APIs menu option](${manage_url}/#/apis) on the left.
+
+> The API tab will already have one API created automatically, the **Auth0 Management API**. For more details on the features of the Management API and its available endpoints, refer to: [Management API](/api/management/v2).
+
+Click the **+ Create API** button.
+
+![Create a new API](/media/articles/api/overview/create-api.png)
+
+You need to provide the following information for your API:
+
+- **Name**: a friendly name for the API. Does not affect any functionality.
+
+- **Identifier**: a unique identifier for the API. We recommend using a URL but note that this doesn't have to be a publicly available URL, Auth0 will not call your API at all. This value **cannot be modified** afterwards.
+
+- **Signing Algorithm**: the algorithm to sign the tokens with. The available values are `HS256` and `RS256`. When selecting `RS256` the token will be signed with the tenant's private key. For more details on the signing algorithms go to the [Signing Algorithms paragraph](#signing-algorithms).
+
+Fill in the required information and click the **Create** button.
+
+Once you do so you will be navigated to the *Quick Start* of your API. Here you can find details on the implementation changes you have to do to your API, which basically consists of choosing a JWT library from a predefined list and configuring this library to validate the access tokens in your API.
+
+![API Quick Starts](/media/articles/api/overview/quickstarts-view.png)
+
+**NOTE**: Keep in mind that we are working on building quickstarts for more stacks, apart from those currently available.
+
+The other available views for your API are:
+
+- **Settings**: lists the settings for your API. Some are editable. Here you can change the token expiration time and enable offline access (this way Auth0 will allow clients to ask for Refresh Tokens for this API).
+
+- **Scopes**: here you can define the scopes for this API, by setting a name and a description.
+
+- **Non Interactive Clients**: lists your Non Interactive Clients. You can authorize which Clients should use your API and which scopes should be granted to each. Only non-interactive Clients require explicit permission. That is because, when you authorize a non-interactive Client to access an API, Auth0 is creating a Client Grant for that Client. For more details on this case refer to: [Setting up a Client Credentials Grant using the Management Dashboard](/api-auth/config/using-the-auth0-dashboard).
```

Comment:
`You can authorize which Clients should use your API and which scopes should be granted to each.`
->
`You can authorize which Non Interactive Clients can request access tokens for your API. You can optionally select a subset of the defined scopes to further limit the access that an authorized client has.`